src/assets/json/games.json

```diff
@@ -0,0 +1,103 @@
+[
+  {
+    "slug": "oot",
+    "title": "Ocarina of Time",
+    "links": {
+      "github": "https://www.github.com/zeldaret/oot",
+      "trello": "https://trello.com/b/qUqnUkWY/ocarina-of-time-decompilation"
+    },
+    "matched": "progress-oot",
+    "unmatched": "progress-oot-u",
+    "charts": [
+      {
+        "title": "Ocarina of Time MQ Debug Rom",
+        "subtitle": "Code decompilation progress",
+        "index": 3,
+        "series": [
+          {
+            "metric": "Code",
+            "description": "{0} / {1} bytes of all code decompiled ({2}%)"
+          },
+          {
+            "metric": "Code",
+            "description": "{0} / {1} bytes of code decompiled ({2}%)"
+          },
+          {
+            "metric": "Boot",
+            "description": "{0} / {1} bytes of boot decompiled ({2}%)"
+          },
+          {
+            "metric": "Overlays",
+            "description": "{0} / {1} bytes of overlay decompiled ({2}%)"
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "slug": "mm",
+    "title": "Majora's Mask",
+    "links": {
+      "github": "https://www.github.com/zeldaret/mm",
+      "trello": "https://trello.com/b/ruxw9n6m/majoras-mask-decompilation"
```

Comment:
Did we still want to link to trello even though we don't use it anymore?

Reply:
I don't think it's useful for outsiders to look at anyway, so I say remove.

Reply:
This can be removed :) note that these files will eventually be pulled from git, so that they can just be put into the assets folder on live

Reply:
Do we want to remove all of the Trello links in that case, or just the MM one?

Reply:
All of them, please.